cli/azd/pkg/input/console.go

```diff
@@ -10,9 +10,11 @@ import (
 	"io"
 	"log"
 	"os"
+	"strconv"
 	"time"
 
 	"github.com/AlecAivazis/survey/v2"
+	"github.com/azure/azure-dev/cli/azd/internal/telemetry/resource"
```

Comment:
nit. Should we move `resource` outside of `telemetry` ?  Just for code hygiene, if `input` is using `resource`, it should be something that is consumed by telemetry and input, and not from telemetry...

Reply:
I do agree with you, but we should wait until we have enough other use cases to form a package. `resource` is fairly separated out from most of `telemetry`. The specific file we're linking to only references another package `fields` which are just string-constants. From an architecture point of view, I don't think we'll run into troubles here. It's purely hygiene based on package name + clean separation, which from a pragmatic standpoint isn't always necessary. 

I'll definitely keep in mind of this in the future.